bindings/tpython/src/TPython.cxx

```diff
@@ -404,7 +399,7 @@ const TPyReturn TPython::Eval(const char *expr)
       return TPyReturn(result);
 
    // explicit conversion for python type required
-   PyObject *pyclass = PyObject_GetAttr(result, PyROOT::PyStrings::gClass);
+   PyObject *pyclass = PyObject_GetAttr(result, CPyCppyy::PyStrings::gBase);
```

Comment:
`gBase` is `"__base__"` and `gClass` is `"__class__"`. Why this change?

Reply:
If the problem is that `"__class__"` is not in the PyStrings of cppyy, you can just do:
```
PyObject *pyclass = PyObject_GetAttrString(result, const_cast<char*>("__class__"));
```